### PR TITLE
Issue201

### PR DESCRIPTION
**Pull request by kdavis-mozilla**
Changed params in prep for a wer-tracking run

### CODE REVIEW

**Comment by kdavis-mozilla** [comment]
@lissyx Could you take a look?

File: bin/run-wer-automation.sh [hunk 1]
```diff
@@ -8,15 +8,27 @@ export ds_dataset_path
 ds_importer="ted"
 export ds_importer
 
-ds_batch_size=32
+ds_batch_size=40
```
Comment (lissyx) [inline bin/run-wer-automation.sh]: I thought 48 was good?

File: bin/run-wer-automation.sh [hunk 2]
```diff
@@ -8,15 +8,27 @@ export ds_dataset_path
...
 export ds_batch_size
 
-ds_training_iters=15
+ds_training_iters=30
```
Comment (lissyx) [inline bin/run-wer-automation.sh]: MOAR COMPUTATION :)

Comment (kdavis-mozilla) [inline bin/run-wer-automation.sh]: :-)

**Pull request merged**